impl/client/src/main/javascript/web/prompting/pentaho-prompting.css

```diff
@@ -1,6 +1,5 @@
 div.prompt-panel {
   padding: 4px 0 0 8px;
-  overflow: auto;
```

Comment:
What's the impact of this change on the other contexts where the Prompting API is used (at least PIR, PDI/DET), besides the report viewer? Also, this is a public API and there may be third-parties depending on it.
So, unless there's a good reason framing this breaking change as generally desirable, this should be opt-in, as was the case of the change done for the report viewer: https://github.com/pentaho/pentaho-platform-plugin-reporting/pull/819/files#diff-18ae6b093acf60b47284d8e074622166f7847f4015e7a1acc4c19041fa075434R40